docs/Actions/inject-data.md

```diff
@@ -0,0 +1,274 @@
+| Name/Value | The first parameter refers to the name of the new token that will be created. The second one refers to the value assigned. The action uses a key-value (name-value) concept. | Yes |
+
+## `Output Parameters Reference`
+
+| Parameter | Description |
+|-----------|-------------|
+| Name | The Name parameter is stored as a token for further use anywhere in the form. |
+
+## `Examples`
+
+### `1. Inject data using Run SQL Query`
+
+​
+The action below selects first username from users database using Run SQL Query and sends it into the form. <br></br>[Import it](/docs/Actions/import-actions.md) into your application to see it in action.
+​
+
+```json
+{
+    "Title": "Execute Actions",
+    "ActionType": "ExecuteActions",
+    "Description": "Extract and inject first username into form",
+    "Condition": null,
+    "Parameters": {
+        "ActionList": [
+            {
+                "Id": -1,
+                "$_uid": "action15911731358816854",
+                "Parameters": {
+                    "ConnectionString": "",
+                    "QueryTimeout": "",
+                    "SqlQuery": "SELECT Username FROM Users\nWHERE UserId = 1",
+                    "BindTokens": [],
+                    "OutputTokenName": "",
+                    "ExtractColumns": [
+                        {
+                            "value": "SQL_Username",
+                            "name": "Username"
+                        }
+                    ],
+                    "OnError": [],
+                    "ShowErrors": false
+                },
+                "ActionType": "RunSql",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Definition": {
+                    "IsClientAction": false,
+                    "Settings": {
+                        "Group": "Data"
+                    }
+                }
+            },
+            {
+                "Id": -1,
+                "$_uid": "action15911731358819956",
+                "Parameters": {
+                    "Data": [
+                        {
+                            "value": "[SQL_Username]",
+                            "name": "Inject_Username"
+                        }
+                    ]
+                },
+                "ActionType": "InjectData",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Definition": {
+                    "IsClientAction": false,
+                    "Settings": {
+                        "Group": "Context"
+                    }
+                }
+            }
+        ]
+    }
+}
+```
+
+### `2. Generate a new token`
+
+​
+The action below generates a new token named "ClientId" with the assigned value "ID15367" that can be used in the form. <br></br> [Import it](/docs/Actions/import-actions.md) into your application to see it in action.
```

Comment:
Will this example work only inside the form?
Also, I'm missing the value. When would I need to do that?
Perhaps, a better example for basic usage would be to inject a token that sets something to Debug=true. Then, add another action, such as Debug Logging, that uses the debug token in a condition.

Reply:
Ok, I updated the example.